src/main/java/com/google/devtools/build/lib/remote/options/RemoteOptions.java

```diff
@@ -139,6 +139,23 @@
       help = "Whether to upload locally executed action results to the remote cache.")
   public boolean remoteUploadLocalResults;
 
+  @Option(
+      name = "incompatible_upload_local_results_disk_only",
+      defaultValue = "false",
+      category = "remote",
+      documentationCategory = OptionDocumentationCategory.REMOTE,
+      effectTags = {OptionEffectTag.UNKNOWN},
+      metadataTags = {
+        OptionMetadataTag.INCOMPATIBLE_CHANGE,
+        OptionMetadataTag.TRIGGERED_BY_ALL_INCOMPATIBLE_CHANGES
+      },
+      help =
+          "If set to true, when --noremote_upload_local_results and "
```

Comment:
Suggested rephrasing:

"If set to true, --noremote_upload_local_results will not apply to the disk cache. If a combined cache is used, --noremote_upload_local_results will cause results to be written to the disk cache, but not uploaded to the remote cache. See #8216 for details."

Reply:
Thanks.